template/src/ontology/Makefile.jinja2

```diff
@@ -133,8 +134,12 @@ show_assets:
 
 # This should be executed by the release manager whenever time comes to make a release.
 # It will ensure that all assets/files are fresh, and will copy to release folder
-prepare_release: $(ASSETS)
-	rsync -R $^ $(RELEASEDIR) &&\
+prepare_release: $(ASSETS) $(PATTERN_RELEASE_FILES)
```

Comment:
I might be inclined to include the pattern release file in ASSETS, but we can do this later.

Reply:
The reason why I didn't (I tried :)) was that that the rsync target dir did not match to the required dir for definitions.owl. definitions.owl needs to be in ./patterns/definitions.owl, but I cant get it there without changing the structure of the rsync! :P But yeah, in future, we can fix it.